lib/release_notes_bot/project_providers.ex

```diff
@@ -11,6 +11,21 @@ defmodule ReleaseNotesBot.ProjectProviders do
     |> Repo.insert()
   end
 
+  def create_default(project_id) do
+    %ProjectProvider{}
+    |> ProjectProvider.changeset(%{
+      project_id: project_id,
+      persistence_provider_id: 1,
```

Comment:
can this be run twice?

Reply:
After some digging - a project could be created twice - but it would be a different record. Right now there is no constraint against duped projects. Could be a good discussion point in the next meeting.